### ISSUE TITLE
Findings in tests not paging properly

### ISSUE THREAD

**Issue by nxc933** [DefectDojo/django-DefectDojo#3939]
**Bug description**
When attempting to view findings from within an engagement in 1.13, clicking a page number or modifying the number of findings per page does not work.

**Steps to reproduce**
1. Go into an engagement
2. Go into a test
3. View findings
4. Click page "2" or modify number of findings per page
5. Issue will occur; page URL will change, but contents of the page does not update

**Expected behavior**
Contents of page will be updated with the requested findings

**Deployment method** *(select with an `X`)*
- [ ] Kubernetes
- [X] Docker
- [ ] setup.bash / legacy-setup.bash

**Environment information**
 - Operating System: Ubuntu 20.04 running in WSL
 - DefectDojo Commit Message: `[2021-02-23 19:54:01 +0100] 3df81273: Merge pull request #3918 from DefectDojo/release/1.13.0 [ (HEAD -> master, tag: 1.13.0, origin/master, origin/HEAD)]`

**Screenshots** (optional)

**Console logs** (optional)
```uwsgi_1         | [pid: 19|app: 0|req: 56/78] 172.20.0.1 () {44 vars in 822 bytes} [Fri Feb 26 15:37:55 2021] GET /test/1?page=2 => generated 377936 bytes in 493 msecs (HTTP/1.1 200) 7 headers in 370 bytes (4 switches on core 1)
nginx_1         | 172.20.0.1 - - [26/Feb/2021:15:37:56 +0000] "GET /test/1?page=2 HTTP/1.1" 200 377936 "http://localhost:8080/test/1?page=4" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:78.0) Gecko/20100101 Firefox/78.0" "-" ```
Result of clicking a link to go to page 2 of findings
In the above case, the URL was modified to http://localhost:8080/test/1?page=2

**Additional context** (optional)
Add any other context about the problem here.

**Comment by valentijnscholten** [comment]
created 2 PRs to fix both in dev and 1.13.x which may get a bugfix release

**Comment by valentijnscholten** [comment]
thanks for reporting
